src/api-handlers/asset/routes/get-assets.route.ts

```diff
@@ -0,0 +1,55 @@
+import { Asset } from '@common/assets/domain/entities/asset';
+import { GetRoute } from '@core/api/route';
+import { Request, RouteHandler } from '@core/api/api.types';
+import { Result } from '@core/architecture/domain/result';
+import { GetAssetsInput } from '../domain/models/get-assets.input';
+import { GetAssetsOutput } from '../domain/models/get-assets.output';
+import { AssetsNotFoundError } from '@common/assets/domain/errors/assets-not-found.error';
+
+/**
+ *
+ * @param {Result<Asset[]>} result
+ * @returns
+ */
+export const parseHandlerResultToResponse = (result: Result<Asset[]>) => {
```

Comment:
Probably we can rename this from Handler to `controller`

Reply:
done